mypy/semanal.py

```diff
@@ -456,6 +461,15 @@ def _visit_func_def(self, defn: FuncDef) -> None:
                     assert ret_type is not None, "Internal error: typing.Coroutine not found"
                     defn.type = defn.type.copy_modified(ret_type=ret_type)
 
+    def prepare_postponed_classmethods_signature(self) -> None:
+        for func in self.postponed_classmethods_stack[-1]:
+            functype = func.type
+            assert self.type
+            assert functype
+            assert isinstance(functype, CallableType)
```

Comment:
Why do you need these asserts? Please add assert messages.

Reply:
That was basically to make `mypy` happy of this code. Fixed to make more sense.